packages/react-dom/src/__tests__/ReactComponent-test.js

```diff
@@ -470,20 +461,11 @@ describe('ReactComponent', () => {
     };
     const element = <div>{[children]}</div>;
     const container = document.createElement('div');
-    let ex;
-    try {
+    expect(() => {
       ReactDOM.render(element, container);
-    } catch (e) {
-      ex = e;
-    }
-    expect(ex).toBeDefined();
-    expect(normalizeCodeLocInfo(ex.message)).toBe(
-      'Objects are not valid as a React child (found: object with keys {x, y, z}).' +
-        (__DEV__
-          ? ' If you meant to render a collection of children, use ' +
-            'an array instead.' +
-            '\n    in div (at **)'
```

Comment:
This one will likely regress full-screen errors in CRA and such, where the extra information ("the above error occurred") isn't shown. That's a problem by itself though.

Reply:
How do those work today? Ours is using a fork so seems like some missing layering.

If implemented as error boundaries, the information is in the componentStack but I assume it needs a global hook?

Reply:
Yeah needs something global.